test/grub2_efi_test.rb

```diff
@@ -4,7 +4,9 @@
 
 describe Bootloader::Grub2EFI do
   before do
-    allow(::CFA::Grub2::Default).to receive(:new).and_return(double("GrubDefault").as_null_object)
+    default = double("GrubDefault").as_null_object
+    allow(default).to receive(:timeout) # collision between timeout method of double and attr
```

Comment:
another possible fix is to use `public_send` instead of `send` which is probably better solution from my POV